std_msgs/Header header
int32 id
float32 dx
float32 dy
float32 dz
float32 fx
float32 fy
float32 fz
bool in_contact
int32 slip_state
